spec/ofx/ofx220_spec.rb

```diff
@@ -64,7 +64,7 @@
         @acct_info.description.should == 'CREDIT CARD'
         @acct_info.bank_id.should == ''
         @acct_info.id.should == '4400720'
-        @acct_info.type.should == nil
+        @acct_info.type.should == :creditcard
```

Comment:
I see this account type should be credit card: https://github.com/maestrano/ofx/blob/feature/account-discovery/spec/fixtures/v220.ofx#L56.

We may, however, want to add a test case where the account type should be nil, as this `null case` is no longer covered in the specs. e.g. if Account Type is unrecognizable?